tools/ci_build/github/azure-pipelines/orttraining-linux-gpu-e2e-test-nightly-pipeline.yml

```diff
@@ -59,23 +59,25 @@ jobs:
     condition: succeededOrFailed() # ensure all tests are run
     timeoutInMinutes: 10
         
-  # Hit OOM with run_training_pipeline_e2e_tests.py - slightly above 16GB limit.
-  # leave this code here for further investigation.
-  # https://msdata.visualstudio.com/Vienna/_workitems/edit/956642
-  # - script: |
-  #     docker run \
-  #       --gpus all \
-  #       --shm-size=1024m \
-  #       --rm \
-  #       --volume $(Build.BinariesDirectory):/build \
-  #       --volume /bert_data:/bert_data \
-  #       --volume /bert_ort:/bert_ort \
-  #       onnxruntime_e2e_test_image \
-  #         /build/RelWithDebInfo/run_training_pipeline_e2e_tests.py \
-  #           --cwd /build/RelWithDebInfo
-  #   displayName: 'Run run_training_pipeline_e2e_tests.py'
-  #   condition: succeededOrFailed() # ensure all tests are run
-  #   timeoutInMinutes: 10
+  - script: |
+      docker run \
+        --gpus all \
+        --shm-size=1024m \
+        --rm \
+        --volume $(Build.BinariesDirectory):/build \
+        --volume /bert_data:/bert_data \
+        onnxruntime_e2e_test_image \
+          mpirun -n 4 -x NCCL_DEBUG=INFO /build/RelWithDebInfo/onnxruntime_training_bert --ort_log_severity 1 --optimizer=Lamb --learning_rate=3e-3 \
```

Comment:
>mpirun -n 4 -x NCCL_DEBUG=INFO /build/RelWithDebInfo/onnxruntime_training_bert --ort_log_severity 1 --optimizer=Lamb --learning_rate=3e-3 \ [](start = 10, length = 139)

most of the command parameter here is the same as the test above. Since the command is so long with various configurations, I am wondering if it is possible to combine those common parameters in one variable and supply the test specific command later (like the one in run_training_pipeline_e2d_tests.py).

If that's not applicable for yml file, maybe we can break the line with one argument per line and add comments for those config that is specific in this test (like data_parallel_size and pipeline_parallel_size)?

Reply:
split into one argument per line to improve readability